### PR TITLE
Add ebook to server download thank you page

### LINKED ISSUE #3283: beta updates: add an ebook to /server download page
- add the ebook partial 'Before you start, you’ll want this eBook' to the /download/server/thank-you page 
- remove this partial from all the other pages it exists on
- update the copy docs

### PR DESCRIPTION
**Pull request by steverydz**
## Done

- Add ebook section to `/download/server/thank-you`
- Remove ebook section from `/openstack/developer`
- Remove ebook section from `/openstack/install`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/thank-you>
- Check that ebook section is on page

## Issue / Card

Fixes #3283 

## Screenshots

screenshot from 2018-06-13 11-32-17

### CODE REVIEW

File: templates/openstack/developer.html [hunk 1]
```diff
@@ -128,6 +128,4 @@ <h2>Need more help?</h2>
   </div>
 </div>
 
-{% include "download/shared/_get_ebook.html"%}
```
Comment (anthonydillon) [inline templates/openstack/developer.html]: The strip above does not need `is-bordered` now.

File: templates/openstack/install.html [hunk 2]
```diff
@@ -216,8 +216,6 @@ <h2>Need more help?</h2>
   </div>
 </div>
 
-{% include "download/shared/_get_ebook.html"%}
```
Comment (anthonydillon) [inline templates/openstack/install.html]: The strip above does not need is-bordered now.

**Comment by steverydz** [comment]
@anthonydillon Fixed

**Comment by pmahnke** [comment]
@steverydz I think this should be right under the top banner with the smilie face... otherwise no one will see it.

**Comment by steverydz** [comment]
@pmahnke Updated

**Comment by steverydz** [comment]
@pmahnke Updated the redirect and fixed the form.

**Comment by pmahnke** [comment]
@steverydz great... what was it?

@anthonydillon must test the form!

**Comment by steverydz** [comment]
@pmahnke There was some js looking for a form that doesn't exist - presumably an old one?

**Pull request merged**